### DISCUSSION TITLE
Use for Mastodon

### DISCUSSION THREAD

**Discussion by outlaw-dame** [BDX-town/Mangane#326] [General]
Is this updated and current enough to be used with Mastodon?

**Comment by Dehelssey** [comment]
Mangane is made to be used with Akkomane's back-end, not Mastodon's.

I think i've heard some people using Mangane with Mastodon but be aware that some features might not work as intended.

**Comment by Cl0v1s** [comment]
Indeed, we are only testing our developments on Akkoma !
